README.md

```diff
@@ -38,49 +38,41 @@ tests. These can be run with,
 esy test
 ```
 
-### Running the node
+### Running a sidechain
 
-The node binary is `http_server.exe` present in `bin/` folder. One way
-to execute it would be,
+#### Requirements
 
-```
-esy b dune exec ./bin/http_server.exe <args>
-```
+To run a sidechain you will need to have a Tezos node that you control and a Tezos Secret Key(edsk).
 
-(`http_server` because the node uses HTTP for p2p communication)
+You can run a Tezos node on a testnet and use the faucet to acquire a key with funds, as described in the following link:
 
-However, every node needs an identity before it can join or start a
-network. Identities can be created with the CLI tool, `sidecli`
+- https://tezos.b9lab.com/tezos-basics/testnet-1
```

Comment:
We could link to the official docs here?
https://tezos.gitlab.io/introduction/test_networks.html#get-free-funds

Reply:
The official docs seems quite crude for people that are not that technical